andela/ah-backend-sparta
# #164798221 Users should be Able to track comment History

#### What does this PR do?
Users can trace edit history

#### Description of Task to be completed?
Users can trace edit history of a comment on an article

#### How should this be manually tested?
fetch `git fetch origin comments-should-track-edit-history-164798221`
change directory `git checkout comments-should-track-edit-history-164798221`
run `pipenv install`
start server `python manage.py runserver`

#### What are the relevant pivotal tracker stories?
[#164798221](https://www.pivotaltracker.com/story/show/164798221)

#### Postman collection
[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/57f841f2bef8323a090b)
